source/Cosmos.System2/Graphics/Canvas.cs

```diff
@@ -429,8 +429,7 @@ protected bool CheckIfModeIsValid(Mode mode)
         {
             Global.mDebugger.SendInternal($"CheckIfModeIsValid");
 
-            if (mode == null)
-                return false;
+            /* Mode has been changed to be a structure so it cannot be null anymore */
```

Comment:
Nit: I think that this comment doesn't make it easier to understand the code, and may create some confusion too.